### PR TITLE
Bump version 2.0.0

### PR DESCRIPTION
**Pull request by kenoss**
Note that we have a changelog from this version.

### CODE REVIEW

**Comment by kenoss** [comment]
I also added URLs to control PyPI page.

File: pyproject.toml [hunk 1]
```diff
@@ -25,6 +25,11 @@ nose2 = "^0.10.0"
 pysen = {version = "^0.9.1", extras = ["lint"]}
 sphinx-theme = "^1.0"
 
+[tool.poetry.urls]
+"Repository" = "https://github.com/Idein/actfw-core"
+"Documentation" = "https://idein.github.io/actfw-core/latest/"
+"Changelog" = "https://github.com/Idein/actfw-core/blob/master/CHANGELOG.md"
```
Comment (laysakura) [inline pyproject.toml:31]: https://python-poetry.org/docs/pyproject/#urls

> If you publish you package on PyPI, they will appear in the Project Links section.

Great!

**Review by laysakura — APPROVED** [review]

**Comment by kenoss** [comment]
Thank you for your review.
2.0.0 must include https://github.com/Idein/actfw-core/pull/14 . So, wait for it a sec.

**Comment by kenoss** [comment]
Closing due to https://github.com/Idein/actfw-core/pull/18 .

**Pull request closed without merging**